### PR TITLE
DISCOVERY: Work Item History

### PR DESCRIPTION
**Pull request by deernano**
Document the research of work item history.

### CODE REVIEW

**Comment by maxandersen** [comment]
[test]

File: ux/discovery/discovery-overview.adoc [hunk 1]
```diff
@@ -1 +1,437 @@
...
+. How the competitive product display history ?
+. What is the using scenario of history and activity ?
+. What is the differences between history and activity ?
+. Where to place the history function ?
+
+
+== Description
+==== *1. Competitive product history function display*
+The research is mainly focusing on 3 competitive products.
+
```
Comment (maxandersen) [inline ux/discovery/discovery-overview.adoc:16]: I would suggest changing Competitive to Similar.

File: ux/discovery/discovery-overview.adoc [hunk 2]
```diff
@@ -1 +1,437 @@
...
+. JIRA
+image:jira.jpg[]
+. asana
+image:asana.png[]
+. taiga
+image:taiga.png[]
+
```
Comment (maxandersen) [inline ux/discovery/discovery-overview.adoc:18]: how about github ?

File: ux/discovery/discovery-overview.adoc [hunk 3]
```diff
@@ -1 +1,437 @@
...
+===== *a. JIRA*
+Using tab to display history
+
+image:product1.png[]
+
+The content display format is:
+
+- avatar / who / what  / when  / content before & after
+- The default is sorted by people.
```
Comment (maxandersen) [inline ux/discovery/discovery-overview.adoc]: if it is sorted by time how can the default be sorted by people ?

File: ux/discovery/discovery-overview.adoc [hunk 4]
```diff
@@ -1 +1,437 @@
...
+= DISCOVERY:Work Item History
+:imagesdir: images
+
+== Overview
+Before we start the wireframe design of the work item history. We need to understand the issue and users’ scenario. The document is to research relevant aspects by asking  the following questions:
+
+. How the competitive product display history ?
+. What is the using scenario of history and activity ?
+. What is the differences between history and activity ?
+. Where to place the history function ?
```
Comment (AdamJ) [inline ux/discovery/discovery-overview.adoc]: Please remove all of the spaces between the end of the text and the question mark.

**Review by AdamJ — COMMENTED** [review]
There are a lot of lines that have spaces between the last line in the sentence and the question mark. These should be removed for consistency, as the majority of the areas have no spaces.

**Comment by aslakknutsen** [comment]
This Discovery seems to have missed one of the 'stakeholders', the government agency agent doing a project/company revision.

**Comment by AdamJ** [comment]
@deernano are you looking to get this merged? If so, can you update per requests?

**Review by kevein — APPROVED** [review]

**Comment by deernano** [comment]
@aslakknutsen "This Discovery seems to have missed one of the 'stakeholders', the government agency agent doing a project/company revision."
Can you explain more about this?

**Comment by alexeykazakov** [comment]
Closing as outdated.

**Pull request closed without merging**